pkgs/servers/monitoring/thanos/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, runCommandNoCC, makeWrapper, buildGoModule, fetchFromGitHub, bazaar }:
+let
+  bazaarNoCertValidation =
+    runCommandNoCC "bzr-no-cert-validation" {
+      inherit bazaar;
+      buildInputs = [ makeWrapper ];
+    } "makeWrapper $bazaar/bin/bzr $out/bin/bzr --add-flags -Ossl.cert_reqs=none";
```

Comment:
is there a way this can go into go-modules instead?

Reply:
We could add it to `nativeBuildInputs` by default but that would cause an unnecessary build dependency for all go-modules (except for `thanos` of course).

However [`git` is also added by default](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/go-modules/generic/default.nix#L39) so there's precedent. OTOH `git` is used by most go-modules while `bzr` isn't. 

In general I think it's worthwhile to have support for overriding go-modules.

Reply:
I understand the need to be able to support overriding the go-modules and I would leave that. I'm just thinking that any package using `bzr` will have to copy your hack over to their package and most likely spend time figuring out why they're getting a TLS error. We could maybe control it with a flag defaulting to false? Something like `enable-bzr ? false`?

Reply:
Apparently `thanos-0.5.0` doesn't depend on `bzr` anymore so I removed the `overrideModAttrs` setting. 

It would still be nice to add a `enableBzr` flag but this can be done in a separate PR.